graph/ids.go

```diff
@@ -39,6 +39,20 @@ func AreSiblingIDs(a, b string) bool {
 	return ParentID(a) == ParentID(b)
 }
 
+// IsNibling checks to see if second is the child of a sibling of the first.
+func IsNibling(fst, snd string) bool {
```

Comment:
yeah, I'll refactor if we still need it after addressing the RootDepthTransform change.